src/Jack.purs

```diff
@@ -8,9 +8,9 @@ module Jack (
   ) where
 
 -- Can't import all 'as X' or we get a warning
-import Jack.Combinators as X0
-import Jack.Gen as X1
-import Jack.Property as X2
-import Jack.Runner as X3
-import Jack.Shrink as X4
-import Jack.Tree as X5
+import Jack.Combinators (arrayOf, arrayOf1, arrayOfN, arrayOfN', boundedChar, boundedInt, chooseChar, chooseInt, elements, frequency, justOf, listOf, listOf1, listOfN, listOfN', maybeOf, noShrink, oneOf, oneOfRec, resize, scale, sized, suchThat, suchThatMaybe) as X0
```

Comment:
In 0.12.x, re-exporting a module without explicitly listing the imports is now a warning. To fix the warning all imports are listed.